### ISSUE TITLE
Error when retrieving user followed artists

### ISSUE THREAD

**Issue by will-lumley** [Peter-Schorn/SpotifyAPI#40]
When I fetch the artists following by the user (on my own personal Spotify account), an error is thrown and the fetch fails.

This is the code that I have:
```
        self.spotifyAPI
            .currentUserFollowedArtists()
            .extendPages(self.spotifyAPI)
            .sink(
                receiveCompletion: { completion in
                    print("Completion: \(completion)")
                },
                receiveValue: { artists in
                    print("Received: \(artists)")
                })
            .store(in: &cancellables)
```

This is the error that's printed to the console:
```
Completion: failure(SpotifyDecodingError: The data from the Spotify web API could not be decoded into 'CursorPagingObject<Artist>'
URL: https://api.spotify.com/v1/me/following?type=artist&after=1hCkSJcXREhrodeIHQdav8&limit=20&locale=en-US,en;q=0.9
http status code: 200
pretty coding path:  (keyNotFound: 'href')
Underlying error:
▿ Swift.DecodingError.keyNotFound
  ▿ keyNotFound: (2 elements)
    - .0: CodingKeys(stringValue: "href", intValue: nil)
    ▿ .1: Swift.DecodingError.Context
      - codingPath: 0 elements
      - debugDescription: "No value associated with key CodingKeys(stringValue: \"href\", intValue: nil) (\"href\")."
      - underlyingError: nil
```

And then this raw data is printed to the console:
```
raw data:
{
  "artists" : {
    "items" : [ {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/1nqY30Z4Bj4NNqZITdGqpX"
      },
      "followers" : {
        "href" : null,
        "total" : 26748
      },
      "genres" : [ "australian indie", "australian pop" ],
      "href" : "https://api.spotify.com/v1/artists/1nqY30Z4Bj4NNqZITdGqpX",
      "id" : "1nqY30Z4Bj4NNqZITdGqpX",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebaa871433dfbdfa7d3c11e356",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174aa871433dfbdfa7d3c11e356",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178aa871433dfbdfa7d3c11e356",
        "width" : 160
      } ],
      "name" : "Ali Barter",
      "popularity" : 27,
      "type" : "artist",
      "uri" : "spotify:artist:1nqY30Z4Bj4NNqZITdGqpX"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/1tt29X4uRBgHBhu9M1vEXM"
      },
      "followers" : {
        "href" : null,
        "total" : 40449
      },
      "genres" : [ "indie pop" ],
      "href" : "https://api.spotify.com/v1/artists/1tt29X4uRBgHBhu9M1vEXM",
      "id" : "1tt29X4uRBgHBhu9M1vEXM",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebd9e9e068fc5975e9f5a3c7e7",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174d9e9e068fc5975e9f5a3c7e7",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178d9e9e068fc5975e9f5a3c7e7",
        "width" : 160
      } ],
      "name" : "Anthony Amorim",
      "popularity" : 44,
      "type" : "artist",
      "uri" : "spotify:artist:1tt29X4uRBgHBhu9M1vEXM"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/1xSSjJrKTO2ZNPU81uLtmI"
      },
      "followers" : {
        "href" : null,
        "total" : 321307
      },
      "genres" : [ "australian hip hop", "australian pop", "pop rap" ],
      "href" : "https://api.spotify.com/v1/artists/1xSSjJrKTO2ZNPU81uLtmI",
      "id" : "1xSSjJrKTO2ZNPU81uLtmI",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebf227a12da7990e1201753f54",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174f227a12da7990e1201753f54",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178f227a12da7990e1201753f54",
        "width" : 160
      } ],
      "name" : "Bliss n Eso",
      "popularity" : 55,
      "type" : "artist",
      "uri" : "spotify:artist:1xSSjJrKTO2ZNPU81uLtmI"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/1zNfnkHqbNqPMm0LY98Tfx"
      },
      "followers" : {
        "href" : null,
        "total" : 26480
      },
      "genres" : [ "alternative emo", "diy emo" ],
      "href" : "https://api.spotify.com/v1/artists/1zNfnkHqbNqPMm0LY98Tfx",
      "id" : "1zNfnkHqbNqPMm0LY98Tfx",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb32ec4ddfdb0df3a6662aa78a",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab6761610000517432ec4ddfdb0df3a6662aa78a",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f17832ec4ddfdb0df3a6662aa78a",
        "width" : 160
      } ],
      "name" : "fredo disco",
      "popularity" : 43,
      "type" : "artist",
      "uri" : "spotify:artist:1zNfnkHqbNqPMm0LY98Tfx"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/20JZFwl6HVl6yg8a4H3ZqK"
      },
      "followers" : {
        "href" : null,
        "total" : 11459957
      },
      "genres" : [ "modern rock" ],
      "href" : "https://api.spotify.com/v1/artists/20JZFwl6HVl6yg8a4H3ZqK",
      "id" : "20JZFwl6HVl6yg8a4H3ZqK",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebb256ae9a4b82bfff97776ae2",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174b256ae9a4b82bfff97776ae2",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178b256ae9a4b82bfff97776ae2",
        "width" : 160
      } ],
      "name" : "Panic! At The Disco",
      "popularity" : 78,
      "type" : "artist",
      "uri" : "spotify:artist:20JZFwl6HVl6yg8a4H3ZqK"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/246dkjvS1zLTtiykXe5h60"
      },
      "followers" : {
        "href" : null,
        "total" : 37500759
      },
      "genres" : [ "dfw rap", "melodic rap", "rap" ],
      "href" : "https://api.spotify.com/v1/artists/246dkjvS1zLTtiykXe5h60",
      "id" : "246dkjvS1zLTtiykXe5h60",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebb894ef9fa437b0389c5567cc",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174b894ef9fa437b0389c5567cc",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178b894ef9fa437b0389c5567cc",
        "width" : 160
      } ],
      "name" : "Post Malone",
      "popularity" : 89,
      "type" : "artist",
      "uri" : "spotify:artist:246dkjvS1zLTtiykXe5h60"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/26aSlbLDsZbZKk9MlapHVj"
      },
      "followers" : {
        "href" : null,
        "total" : 6691
      },
      "genres" : [ ],
      "href" : "https://api.spotify.com/v1/artists/26aSlbLDsZbZKk9MlapHVj",
      "id" : "26aSlbLDsZbZKk9MlapHVj",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebe28dc967e1f27fdc0295d6d4",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174e28dc967e1f27fdc0295d6d4",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178e28dc967e1f27fdc0295d6d4",
        "width" : 160
      } ],
      "name" : "Nick Weaver",
      "popularity" : 22,
      "type" : "artist",
      "uri" : "spotify:artist:26aSlbLDsZbZKk9MlapHVj"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2DORQjKJVYZMx9uu82UGtT"
      },
      "followers" : {
        "href" : null,
        "total" : 385626
      },
      "genres" : [ "australian pop", "australian talent show" ],
      "href" : "https://api.spotify.com/v1/artists/2DORQjKJVYZMx9uu82UGtT",
      "id" : "2DORQjKJVYZMx9uu82UGtT",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebd021d5a283c0d3c935f4bb3f",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174d021d5a283c0d3c935f4bb3f",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178d021d5a283c0d3c935f4bb3f",
        "width" : 160
      } ],
      "name" : "Amy Shark",
      "popularity" : 57,
      "type" : "artist",
      "uri" : "spotify:artist:2DORQjKJVYZMx9uu82UGtT"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2JY5qzEozvTdogkDTkkOMf"
      },
      "followers" : {
        "href" : null,
        "total" : 182561
      },
      "genres" : [ "australian children's music", "children's music" ],
      "href" : "https://api.spotify.com/v1/artists/2JY5qzEozvTdogkDTkkOMf",
      "id" : "2JY5qzEozvTdogkDTkkOMf",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb0496588a3aaa99244ddfd6e8",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab676161000051740496588a3aaa99244ddfd6e8",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f1780496588a3aaa99244ddfd6e8",
        "width" : 160
      } ],
      "name" : "The Wiggles",
      "popularity" : 65,
      "type" : "artist",
      "uri" : "spotify:artist:2JY5qzEozvTdogkDTkkOMf"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2QxpVKPoV0NsJrzrrPQitO"
      },
      "followers" : {
        "href" : null,
        "total" : 12282
      },
      "genres" : [ "australian underground hip hop" ],
      "href" : "https://api.spotify.com/v1/artists/2QxpVKPoV0NsJrzrrPQitO",
      "id" : "2QxpVKPoV0NsJrzrrPQitO",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab67616d0000b2735066e724b80c80e743baf9b0",
        "width" : 640
      }, {
        "height" : 300,
        "url" : "https://i.scdn.co/image/ab67616d00001e025066e724b80c80e743baf9b0",
        "width" : 300
      }, {
        "height" : 64,
        "url" : "https://i.scdn.co/image/ab67616d000048515066e724b80c80e743baf9b0",
        "width" : 64
      } ],
      "name" : "TZU",
      "popularity" : 25,
      "type" : "artist",
      "uri" : "spotify:artist:2QxpVKPoV0NsJrzrrPQitO"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2Sqr0DXoaYABbjBo9HaMkM"
      },
      "followers" : {
        "href" : null,
        "total" : 1812617
      },
      "genres" : [ "acoustic pop", "dance pop", "hollywood", "lilith", "neo mellow", "pop", "pop rock" ],
      "href" : "https://api.spotify.com/v1/artists/2Sqr0DXoaYABbjBo9HaMkM",
      "id" : "2Sqr0DXoaYABbjBo9HaMkM",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb0bae7cfd3b32b10154e0b8b3",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab676161000051740bae7cfd3b32b10154e0b8b3",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f1780bae7cfd3b32b10154e0b8b3",
        "width" : 160
      } ],
      "name" : "Sara Bareilles",
      "popularity" : 65,
      "type" : "artist",
      "uri" : "spotify:artist:2Sqr0DXoaYABbjBo9HaMkM"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2Waw2sSbqvAwK8NwACNjVo"
      },
      "followers" : {
        "href" : null,
        "total" : 1522885
      },
      "genres" : [ "comic" ],
      "href" : "https://api.spotify.com/v1/artists/2Waw2sSbqvAwK8NwACNjVo",
      "id" : "2Waw2sSbqvAwK8NwACNjVo",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb30d9a4acdf8cd3e8c0ad39ab",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab6761610000517430d9a4acdf8cd3e8c0ad39ab",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f17830d9a4acdf8cd3e8c0ad39ab",
        "width" : 160
      } ],
      "name" : "Bo Burnham",
      "popularity" : 74,
      "type" : "artist",
      "uri" : "spotify:artist:2Waw2sSbqvAwK8NwACNjVo"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2YZyLoL8N0Wb9xBt1NhZWg"
      },
      "followers" : {
        "href" : null,
        "total" : 20931140
      },
      "genres" : [ "conscious hip hop", "hip hop", "rap", "west coast rap" ],
      "href" : "https://api.spotify.com/v1/artists/2YZyLoL8N0Wb9xBt1NhZWg",
      "id" : "2YZyLoL8N0Wb9xBt1NhZWg",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb437b9e2a82505b3d93ff1022",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174437b9e2a82505b3d93ff1022",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178437b9e2a82505b3d93ff1022",
        "width" : 160
      } ],
      "name" : "Kendrick Lamar",
      "popularity" : 89,
      "type" : "artist",
      "uri" : "spotify:artist:2YZyLoL8N0Wb9xBt1NhZWg"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2ZGFZCuhX2xH6atSNq2rWN"
      },
      "followers" : {
        "href" : null,
        "total" : 998
      },
      "genres" : [ "kc indie" ],
      "href" : "https://api.spotify.com/v1/artists/2ZGFZCuhX2xH6atSNq2rWN",
      "id" : "2ZGFZCuhX2xH6atSNq2rWN",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb300c9624306bffde7d679539",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174300c9624306bffde7d679539",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178300c9624306bffde7d679539",
        "width" : 160
      } ],
      "name" : "Rachel Mallin & the Wild Type",
      "popularity" : 3,
      "type" : "artist",
      "uri" : "spotify:artist:2ZGFZCuhX2xH6atSNq2rWN"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2eBmUbHKMQCAdD824cSiIL"
      },
      "followers" : {
        "href" : null,
        "total" : 34047
      },
      "genres" : [ "math rock" ],
      "href" : "https://api.spotify.com/v1/artists/2eBmUbHKMQCAdD824cSiIL",
      "id" : "2eBmUbHKMQCAdD824cSiIL",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebf3bc6d0956cba839bcace259",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174f3bc6d0956cba839bcace259",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178f3bc6d0956cba839bcace259",
        "width" : 160
      } ],
      "name" : "Feed Me Jack",
      "popularity" : 40,
      "type" : "artist",
      "uri" : "spotify:artist:2eBmUbHKMQCAdD824cSiIL"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2h93pZq0e7k5yf4dywlkpM"
      },
      "followers" : {
        "href" : null,
        "total" : 9677148
      },
      "genres" : [ "alternative r&b", "hip hop", "lgbtq+ hip hop", "neo soul", "pop" ],
      "href" : "https://api.spotify.com/v1/artists/2h93pZq0e7k5yf4dywlkpM",
      "id" : "2h93pZq0e7k5yf4dywlkpM",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebfbc3faec4a370d8393bee7f1",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174fbc3faec4a370d8393bee7f1",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178fbc3faec4a370d8393bee7f1",
        "width" : 160
      } ],
      "name" : "Frank Ocean",
      "popularity" : 81,
      "type" : "artist",
      "uri" : "spotify:artist:2h93pZq0e7k5yf4dywlkpM"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2hR4h1Cao2ueuI7Cx9c7V8"
      },
      "followers" : {
        "href" : null,
        "total" : 3272310
      },
      "genres" : [ "bedroom pop", "indie pop" ],
      "href" : "https://api.spotify.com/v1/artists/2hR4h1Cao2ueuI7Cx9c7V8",
      "id" : "2hR4h1Cao2ueuI7Cx9c7V8",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb7b3e522a30dfef887aad0eef",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab676161000051747b3e522a30dfef887aad0eef",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f1787b3e522a30dfef887aad0eef",
        "width" : 160
      } ],
      "name" : "Cavetown",
      "popularity" : 73,
      "type" : "artist",
      "uri" : "spotify:artist:2hR4h1Cao2ueuI7Cx9c7V8"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/2vnB6tuQMaQpORiRdvXF9H"
      },
      "followers" : {
        "href" : null,
        "total" : 941283
      },
      "genres" : [ "bubblegrunge", "chicago indie", "indie pop" ],
      "href" : "https://api.spotify.com/v1/artists/2vnB6tuQMaQpORiRdvXF9H",
      "id" : "2vnB6tuQMaQpORiRdvXF9H",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebbf42b607fb398365296230b0",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174bf42b607fb398365296230b0",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178bf42b607fb398365296230b0",
        "width" : 160
      } ],
      "name" : "Beach Bunny",
      "popularity" : 69,
      "type" : "artist",
      "uri" : "spotify:artist:2vnB6tuQMaQpORiRdvXF9H"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/37J1PlAkhRK7yrZUtqaUpQ"
      },
      "followers" : {
        "href" : null,
        "total" : 921441
      },
      "genres" : [ "electro swing", "nu jazz" ],
      "href" : "https://api.spotify.com/v1/artists/37J1PlAkhRK7yrZUtqaUpQ",
      "id" : "37J1PlAkhRK7yrZUtqaUpQ",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5ebdedbe7030797b73b6a77e39f",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab67616100005174dedbe7030797b73b6a77e39f",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f178dedbe7030797b73b6a77e39f",
        "width" : 160
      } ],
      "name" : "Caravan Palace",
      "popularity" : 59,
      "type" : "artist",
      "uri" : "spotify:artist:37J1PlAkhRK7yrZUtqaUpQ"
    }, {
      "external_urls" : {
        "spotify" : "https://open.spotify.com/artist/395RUhnuJcbFdgp6nYZsih"
      },
      "followers" : {
        "href" : null,
        "total" : 162561
      },
      "genres" : [ "lo-fi rap" ],
      "href" : "https://api.spotify.com/v1/artists/395RUhnuJcbFdgp6nYZsih",
      "id" : "395RUhnuJcbFdgp6nYZsih",
      "images" : [ {
        "height" : 640,
        "url" : "https://i.scdn.co/image/ab6761610000e5eb34023fa2a0933f8acfc99031",
        "width" : 640
      }, {
        "height" : 320,
        "url" : "https://i.scdn.co/image/ab6761610000517434023fa2a0933f8acfc99031",
        "width" : 320
      }, {
        "height" : 160,
        "url" : "https://i.scdn.co/image/ab6761610000f17834023fa2a0933f8acfc99031",
        "width" : 160
      } ],
      "name" : "atlas",
      "popularity" : 49,
      "type" : "artist",
      "uri" : "spotify:artist:395RUhnuJcbFdgp6nYZsih"
    } ],
    "next" : "https://api.spotify.com/v1/me/following?type=artist&after=395RUhnuJcbFdgp6nYZsih&limit=20&locale=en-US,en",
    "total" : 103,
    "cursors" : {
      "after" : "395RUhnuJcbFdgp6nYZsih"
    },
    "limit" : 20,
    "href" : "https://api.spotify.com/v1/me/following?type=artist&after=1hCkSJcXREhrodeIHQdav8&limit=20&locale=en-US,en"
  }
})
```

**Comment by Peter-Schorn** [comment]
I've determined the source of the bug. At this time, If more than one page is returned, subsequent pages cannot be decoded by the `extendPages` method. Expect a fix shortly.

**Comment by Peter-Schorn** [comment]
Fixed in 32ff6b5448a3efe10f5587f4d7311f694ff51838 (2.1.1).
